### ISSUE TITLE
Setting/overwriting env/path externally in prefs file?

### ISSUE THREAD

**Issue by og2t** [Swoffa/SublimeKickAssemblerC64#22]
Hi Swoffa,

Would it be possible to set/overwrite env/path settings outside, i.e. via the prefs file?
On my work machine all my C64 related stuff lives in the mountable virtual drive and I need to overwrite these paths somehow. Cheers!

```
"osx":
    {
        "env" : { "CLASSPATH":"$CLASSPATH:/Applications/KickAssembler/KickAss.jar"},
        "path": "$PATH:/Applications/VICE/X64.app/Contents/MacOS/"
    },
```

**Comment by Swoffa** [comment]
Added `kickass_path` setting for adding things to path. Env might be fixed later, but you can use the `kickass_jar_path` setting for the kick assembler jar path (full path to the Kick Assembler jar-file).

There is also setting for Vice, `kickass_run_path` and `kickass_debug_path`.

More info (under settings): https://goatpower.org/projects-releases/sublime-package-kick-assembler-c64/

**Comment by Swoffa** [comment]
Fixed in pr #40 

Now environment variables can be added thru the `kickass_env` setting.
The variables defined will have precedence over other environment variables with the same names.
Example setting:
```
	"kickass_env": { 
		"MY_ENV_VAR1" : "Goats can fly",
		"MY_ENV_VAR2" : "Brown banana",
	}
```

Hope it helps!
